docker-compose.yml

```diff
@@ -1,17 +1,18 @@
-version: '3'
+version: '2'
 
 services:
   db:
-    image: postgres
+    image: postgres:alpine
     volumes:
-      - ./sql_init.sh:/docker-entrypoint-initdb.d/init_user_db.sh
-      - ./postgres-data:/var/lib/postgresql/data
+      - './sql_init.sh:/docker-entrypoint-initdb.d/init_user_db.sh:ro'
+      - 'pg_data:/var/lib/postgresql/data'
+
```

Comment:
Here you're also renaming things for no reason.

Reply:
Here i am changing nothing, I have switched it to use a docker volume instead of creating a dir owned by root within my home dir.